src/librustdoc/formats/cache.rs

```diff
@@ -24,7 +24,7 @@ use crate::html::render::IndexItem;
 /// to be a fairly large and expensive structure to clone. Instead this adheres
 /// to `Send` so it may be stored in an `Arc` instance and shared among the various
 /// rendering threads.
-#[derive(Default)]
+#[derive(Default, Debug)]
```

Comment:
Same question as above

Reply:
it  seems search_index.rs need this derive Debug.
the error message is list as below without Debug :
error[E0277]: `Cache` doesn't implement `Debug`
   --> src/librustdoc/html/render/search_index.rs:240:1
    |
240 |   #[instrument(level = "trace", skip(tcx, res))]
    |   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |   |
    |   `Cache` cannot be formatted using `{:?}`
    |   in this procedural macro expansion
    |
   ::: /home/zredb/.cargo/registry/src/mirrors.ustc.edu.cn-61ef6e0cd06fb9b8/tracing-attributes-0.1.18/src/lib.rs:459:1
    |
459 | / pub fn instrument(
460 | |     args: proc_macro::TokenStream,
461 | |     item: proc_macro::TokenStream,
462 | | ) -> proc_macro::TokenStream {
    | |____________________________- in this expansion of `#[instrument]`
    |
    = help: the trait `Debug` is not implemented for `Cache`
    = note: add `#[derive(Debug)]` to `Cache` or manually `impl Debug for Cache`
    = note: required because of the requirements on the impl of `Debug` for `&Cache`
    = note: 1 redundant requirement hidden
    = note: required because of the requirements on the impl of `Debug` for `&&Cache`
note: required by a bound in `tracing::field::debug`
   --> /home/zredb/.cargo/registry/src/mirrors.ustc.edu.cn-61ef6e0cd06fb9b8/tracing-core-0.1.21/src/field.rs:265:8
    |
265 |     T: fmt::Debug,
    |        ^^^^^^^^^^ required by this bound in `tracing::field::debug`